# HG changeset patch
# User Rik <rik@octave.org>
# Date 1457190034 28800
#      Sat Mar 05 07:00:34 2016 -0800
# Node ID 1f69fc65539ee750dba0cb10d87ffae3121bcb09
# Parent  120bb822c6f4a30217a174f40d28da2b21854976
maint: Correct typo in comments of cset 9429c39994f8.

* subst-config-vals.in.sh, subst-cross-config-vals.in.sh: Replace "on" with "one".

diff --git a/build-aux/subst-config-vals.in.sh b/build-aux/subst-config-vals.in.sh
--- a/build-aux/subst-config-vals.in.sh
+++ b/build-aux/subst-config-vals.in.sh
@@ -175,17 +175,17 @@ X11_INCFLAGS='@X11_INCFLAGS@'
 X11_LIBS='@X11_LIBS@'
 XTRA_CFLAGS='@XTRA_CFLAGS@'
 XTRA_CXXFLAGS='@XTRA_CXXFLAGS@'
 YACC='@YACC@'
 YFLAGS='@YFLAGS@'
 Z_CPPFLAGS='@Z_CPPFLAGS@'
 Z_LDFLAGS='@Z_LDFLAGS@'
 Z_LIBS='@Z_LIBS@'
-## this on uses ' to quote individual arguments, so use " here.
+## this one uses ' to quote individual arguments, so use " here.
 config_opts="@config_opts@"
 
 ## These must use " so that embedded variables are interpolated.
 
 ## Is there a better way?
 
 function expand_var ()
 {
diff --git a/build-aux/subst-cross-config-vals.in.sh b/build-aux/subst-cross-config-vals.in.sh
--- a/build-aux/subst-cross-config-vals.in.sh
+++ b/build-aux/subst-cross-config-vals.in.sh
@@ -179,17 +179,17 @@ X11_INCFLAGS='@X11_INCFLAGS@'
 X11_LIBS='@X11_LIBS@'
 XTRA_CFLAGS='@XTRA_CFLAGS@'
 XTRA_CXXFLAGS='@XTRA_CXXFLAGS@'
 YACC='@YACC@'
 YFLAGS='@YFLAGS@'
 Z_CPPFLAGS='@Z_CPPFLAGS@'
 Z_LDFLAGS='@Z_LDFLAGS@'
 Z_LIBS='@Z_LIBS@'
-## this on uses ' to quote individual arguments, so use " here.
+## this one uses ' to quote individual arguments, so use " here.
 config_opts="@config_opts@"
 
 ## These must use " so that embedded variables are interpolated.
 
 ## Is there a better way?
 
 function expand_var ()
 {
